style/amenity-points.mss

```diff
@@ -3028,6 +3028,38 @@
       }
     }
   }
+
+  [feature = 'man_made_goods_conveyor'] {
+    [zoom >= 12] {
```

Comment:
Have you tested this at z12 and z13? Are there long enough goods_conveyor features to be shown clearly at z12? With this pattern the goods_conveyor will need to be at least 40 pixels long for the pattern to be understood.

Reply:
Indeed, z14 is suitable in most cases but there are rare large scale features requiring z12 to z13. What do you suggest ?

https://www.openstreetmap.org/relation/2606647
z12
![conveyor4_z12](https://user-images.githubusercontent.com/9897203/78669463-953a8500-78dc-11ea-8114-73ad2448de35.png)

z13
![conveyor4_z13](https://user-images.githubusercontent.com/9897203/78669473-9a97cf80-78dc-11ea-8468-fbd42591cdde.png)

z14
![conveyor4_z14](https://user-images.githubusercontent.com/9897203/78669481-9f5c8380-78dc-11ea-82da-6190b9ad46be.png)

Reply:
In these images, at z12 the current rendering is too strong: it is much wider than most types of highways and other features, so it would need to be a thinner version.

But even then, in this example many of the goods_conveyor ways are very close to overlapping at z12. By z14 this is not a problem. Most of these ways are several kilometers long

I downloaded all man_made=goods_conveyor ways, and checked their length in JOSM. Only 370 out of 4324 are longer than 2 kilometers, which is where they are long enough to see clearly at z12. 3130 are less than 500 meters long, and the median length is only 200 meters.

So I would recommend starting the rendering at z14.